raiden/network/rpc/client.py

```diff
@@ -187,8 +187,8 @@ def new_filter(jsonrpc_client, contract_address, topics, from_block=None, to_blo
     if isinstance(to_block, int):
         to_block = hex(to_block)
     json_data = {
-        'fromBlock': from_block if from_block is not None else 'latest',
```

Comment:
We have to be explicit check for None here, since some times it may somehow end up being `''` and in that case we also want the `else` branch.